src/SendToGovNotifyHandler.ts

```diff
@@ -1,14 +1,21 @@
+/* eslint-disable complexity */
+import { Context  } from "aws-lambda";
 import { Logger } from "@aws-lambda-powertools/logger";
 import { Metrics } from "@aws-lambda-powertools/metrics";
 import { LambdaInterface } from "@aws-lambda-powertools/commons";
 import { Constants } from "./utils/Constants";
+import { SendToGovNotifyProcessor } from "./services/SendToGovNotifyProcessor";
+import { getParameter } from "./utils/Config";
+import { EnvironmentVariables } from "./services/EnvironmentVariables";
+import { ServicesEnum } from "./models/enums/ServicesEnum";
+import { MessageCodes } from "./models/enums/MessageCodes";
+import { AppError } from "./utils/AppError";
+import { HttpCodesEnum } from "./utils/HttpCodesEnum";
 
-const {
-	POWERTOOLS_METRICS_NAMESPACE = Constants.F2F_METRICS_NAMESPACE,
-	POWERTOOLS_LOG_LEVEL = Constants.DEBUG,
-	POWERTOOLS_SERVICE_NAME = "TODO",
-} = process.env;
 
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : Constants.EMAIL_METRICS_NAMESPACE;
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : Constants.DEBUG;
```

Comment:
Can we change the default here to info so this doesnt log debug logs if this is ever changed by mistake in a higher environment